dashboard/public/js/tutorial.js

```diff
@@ -173,7 +173,18 @@ function sugarizerTour(currentView, role, mode) {
 			tour.addStep(getStep("classroom", "#classroom-delete-class", "left", 8));
 			tour.addStep(getStep("classroom", "#checkAll", "right", 9));
 			tour.addStep(getStep("classroom", "#classroom-deleteMultiple", "left", 10));
-		} else if (currentView == "stats") {
+		} else if (currentView == "assignment") {
+			tour.addStep(getStep("assignment", "#assignment-serach-row", "bottom", 2));
+			tour.addStep(getStep("assignment", "#assignment-deleteMultiple", "left", 10));
+			tour.addStep(getStep("assignment", "#assignment-addassignment", "left", 3));
+			tour.addStep(getStep("assignment", "#assignment-cards-parent", "top", 4));
+			tour.addStep(getStep("assignment", "#checkAll", "right", 9));
+			tour.addStep(getStep("assignment", "#assignment-card", "bottom", 5));
+			tour.addStep(getStep("assignment", "#assignment-delete-class", "left", 8));
+			tour.addStep(getStep("assignment", "#assignment-view-students", "left", 6));
+			tour.addStep(getStep("assignment", "#assignment-launch", "left", 5));
+		}
```

Comment:
Tutorial can also be added for deliveries view.

Reply:
Done ! also fix tuts for assignment view